features/netsocket/mbed_lib.json

```diff
@@ -65,6 +65,10 @@
         "socket-stats-max-count": {
             "help": "Maximum number of socket statistics cached",
             "value": 10
+        },
+        "offload-tlssocket" : {
```

Comment:
Just a thought here, please reject if "offload" is in fact a widely used name for what you are trying to achieve :) 
To me it sounds like "offload" is a potential gain for the user, as they offload the memory and processor (and probably increase the load on the external modem?) but doesn't exactly indicate the main choice that user makes with this flag.
I wonder if something like "external" (from mbed-os perspective) or "embedded" would better suit the purpose? 
Are there any reasons, different than offloading the main core, that would make users switch this on?

Reply:
@AnttiKauppila , @kjbracey-arm, @lauri-piikivi  what do you think about the term "offloaded"? The same term was already used for DNS: https://github.com/ARMmbed/mbed-os/blob/4e10e9896dfd0eef6e61c3a5bdbd7564927f5162/features/cellular/mbed_lib.json#L24